src/igvfd/audit/measurement_set.py

```diff
@@ -492,6 +492,35 @@ def audit_missing_auxiliary_set(value, system):
                 yield AuditFailure(audit_message.get('audit_category', ''), f'{detail} {audit_message.get("audit_description", "")}', level=audit_message.get('audit_level', ''))
 
 
+@audit_checker('MeasurementSet', frame='embedded')
```

Comment:
It doesn't seem like this needs frame=embedded - was the test failing if it was not embedded?

Reply:
I should try that.